builtins/src/string.rs

```diff
@@ -442,11 +442,11 @@ Make a new string by applying lambda to each char in input string.
 Section: string
 
 Example:
-(test::assert-equal "XstringXstrX" (str-map "xstringxstrx" (fn (ch) (if (= \x ch) \X ch))))
-(def test-str-map (str-map "xstringxstrx" (fn (ch) (if (equal? \x ch) \X ch))))
+(test::assert-equal "XstringXstrX" (str-map "xstringxstrx" (fn (ch) (if (equal? "x" ch) "X" ch))))
+(def test-str-map (str-map "xstringxstrx" (fn (ch) (if (equal? "x" ch) "X" ch))))
```

Comment:
is there a syntax to get the characters x or X?

Reply:
Not sure I follow but "x" is not a char in slosh (or any lisp I know of).  It used to be \x and this was how you read char 'x'.